src/hammer-vlsi/defaults.yml

```diff
@@ -276,6 +276,7 @@ vlsi.inputs:
   # how many domains you have and what their relation is.
   # Valid options:
   # auto - Hammer will generate a power specification based on other settings.
+  #   Currently the other settings include at least vlsi.inputs.supplies.
```

Comment:
What do you think about something like "Hammer will generate a power specification based on other settings, including vlsi.inputs.supplies"? I had to think a bit about "Currently the other settings include" on first glance.

Reply:
Sure go for it.